environment.yml

```diff
@@ -0,0 +1,22 @@
+# YML file for setting up the Digital-Twin Python environment
+
+name: digitaltwin
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - pip
+  - python-dotenv
+  - geopandas>=0.10.2 # Needs to be explicit otherwise the dependencies required to use sjoin are not installed
+  - rasterio # Needs to be explicit as wheel fails to build if insalled as a dependency of geofabrics
+  - python-pdal # Not on PyPI so needs to be added here and not through pip geofabrics
+  - geoapis>=0.2.8
+  - psycopg2 # Try remove in future
+  - bs4 # Try remove in future
+  - validators  # Try remove in future
+  - sqlalchemy
+  - geoalchemy2
+  - pip:
```

Comment:
I think you missed pytest, otherwise good to go.

Reply:
good point! Added now